opennav_following_bt/include/opennav_following_bt/follow_object.hpp

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2024 Open Navigation LLC
+// Copyright (c) 2024 Alberto J. Tudela Roldán
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OPENNAV_FOLLOWING_BT__FOLLOW_OBJECT_HPP_
+#define OPENNAV_FOLLOWING_BT__FOLLOW_OBJECT_HPP_
+
+#include <string>
+#include <memory>
+#include <vector>
+
+#include "opennav_following_msgs/action/follow_object.hpp"
+#include "geometry_msgs/msg/pose_stamped.hpp"
+#include "nav2_behavior_tree/bt_action_node.hpp"
+
+namespace opennav_following_bt
+{
+
+/**
+ * @brief nav2_behavior_tree::BtActionNode class that wraps opennav_following_msgs/FollowObject
+ */
+class FollowObjectAction
```

Comment:
I think a Cancel for this BT node would be good so that users can stop once they're completed